crates/turbopack-css/src/module_asset.rs

```diff
@@ -193,21 +197,37 @@ impl EcmascriptChunkItem for ModuleChunkItem {
                 }
                 code += "});\n";
                 EcmascriptChunkItemContent {
-                    inner_code: code.into(),
-                    // TODO: We generate a minimal map for runtime code so that the filename is
+                    inner_code: code.clone().into(),
+                    // We generate a minimal map for runtime code so that the filename is
                     // displayed in dev tools.
+                    source_map: Some(generate_minimal_source_map(
+                        format!("{}.js", self.module.path().await?.path),
+                        code,
+                    )),
                     ..Default::default()
                 }
             }
             ParseResult::NotFound | ParseResult::Unparseable => {
+                let code = "__turbopack_export_value__({});\n";
                 EcmascriptChunkItemContent {
-                    inner_code: "__turbopack_export_value__({});\n".into(),
-                    // TODO: We generate a minimal map for runtime code so that the filename is
+                    inner_code: code.into(),
+                    // We generate a minimal map for runtime code so that the filename is
                     // displayed in dev tools.
+                    source_map: Some(generate_minimal_source_map(
+                        format!("{}.js", self.module.path().await?.path),
+                        code.into(),
+                    )),
                     ..Default::default()
                 }
             }
         }
         .cell())
     }
 }
+
+fn generate_minimal_source_map(filename: String, source: String) -> ParseResultSourceMapVc {
+    let sm: Arc<SourceMap> = Default::default();
+    sm.new_source_file(FileName::Custom(filename), source);
```

Comment:
Since this isn't really a source map, could this just be an empty `source` string?

Reply:
There will be a error displayed in the chrome's dev tools, maybe we can leave a message as a placeholder of the `source` string, any suggestions for the message?
<img width="1144" alt="Screen Shot 2023-01-16 at 10 43 40" src="https://user-images.githubusercontent.com/42857895/212587465-f597c11f-66b9-43ff-b28e-f8906b96ffb9.png">

Reply:
Actually, I guess this is fine.